docs/pages/guides/tailwind.mdx

```diff
@@ -58,53 +55,43 @@ Install `tailwindcss` and its required peer dependencies. Then, the run initiali
 
 <Step label="2">
 
-Add paths to all of your template files inside **tailwind.config.js**.
-
-```js tailwind.config.js
-/** @type {import('tailwindcss').Config} */
-module.exports = {
-  content: [
-    // Ensure this points to your source code
-    './app/**/*.{js,tsx,ts,jsx}',
-    // If you use a `src` directory, add: './src/**/*.{js,tsx,ts,jsx}'
-    // Do the same with `components`, `hooks`, `styles`, or any other top-level directories
-  ],
-  theme: {
-    extend: {},
+Add Tailwind to your PostCSS configuration
+
+```js postcss.config.js
+export default {
+  plugins: {
+    '@tailwindcss/postcss': {},
   },
-  plugins: [],
 };
 ```
 
-> If you are using Expo Router, consider using a root **src** directory to simplify this step. Learn more about [top-level src directory](/router/reference/src-directory/).
-
 </Step>
 
 <Step label="3">
 
-Create a **global.css** file in the root of your project and directives for each of Tailwind's layers:
+Create a CSS file that imports Tailwind CSS.
 
 ```css global.css
-/* This file adds the requisite utility classes for Tailwind to work. */
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
 ```
 
+> **info** You can choose any name for this file. Using `global.css` is common practice.
```

Comment:
```suggestion
> **info** You can choose any name for this file. Using **global.css** is common practice.
```

- I think, instead of calling this out in a callout, we can move this with the text of this step.
- Let's also make filename bold as per our writing stylegyuide.